### DISCUSSION TITLE
Domínio pylestras.org

### DISCUSSION THREAD

**Discussion by alynnefs** [apyb/comunidade#44] [Pergunta]
O domínio http://pylestras.org era usado pelo PUG-CE, mas expirou e não pretendem renovar. Acham que é interessante manter pra algo a nível nacional ou liberamos?

cc @fabiocerqueira, @italomaia

**Comment by tiidadavena** [comment]
@alynnefs a demanda ainda permanece relevante para o PUG-CE?

**Reply by alynnefs** [reply]
Acredito que não, mas não olhei se já "perdemos" o domínio

**Reply by betosales** [reply]
o domínio esta disponível para compra, olhei a pouco no godaddy e ta disponível por R$54,99 no primeiro ano, mas visto que a pergunta era de 2022...
